### ISSUE TITLE
graphite timestamp float implementation: is this correct/required?

### ISSUE THREAD

**Issue by inge4pres** [criteo/graphite-remote-adapter#44]
The current implementation only supports float timestamp with default precision for the data sent to graphite. AFAIK not all graphite implementations support float timestamp, nor such a high precision is usually required; our workaround is to use %.0f in https://github.com/criteo/graphite-remote-adapter/blob/master/client/graphite/write.go#L40

Would it be ok to configure the data point resolution based on a parameter? I.E. we could have

```
func (c *Client) prepareDataPoint(path string, s *model.Sample, subSecondEnabled bool) string {
...
}
```

Thank you

**Comment by iksaif** [comment]
I think most of Graphite *does not* support float precision. We should send the timestamp as an integer only

**Comment by inge4pres** [comment]
As most Graphite will be backed by Whisper and this does not support sub-second resolution (see http://graphite.readthedocs.io/en/latest/whisper.html#data-points) this has been resolved merging #43 and specifically commit https://github.com/criteo/graphite-remote-adapter/commit/006480b714e891eb2aab5bbc678bfe6025c3285e
